queries/usersQueries.js

```diff
@@ -1,18 +1,31 @@
-const usersData = require('../usersData.json');
+// const usersData = require("../usersData.json");
+const db = require("../db/index");
 
-const idToUserIndex = usersData.reduce((index, user) => {
-  const { id } = user;
-  index[id] = user;
-  return index;
-}, {});
+// const idToUserIndex = usersData.reduce((index, user) => {
+//   const { id } = user;
+//   index[id] = user;
+//   return index;
+// }, {});
 
-const getAllUsers = () => usersData;
+async function getAllUsers() {
+  try {
+    const users = await db.any("SELECT * FROM users");
+    return users;
+  } catch (error) {
+    console.log(error);
```

Comment:
If you catch the error here, you need to get it back to the controller somehow so that it knows how to handle the error. Console.log will "swallow" the error and the controller will not be able to return the message to the client.